assemblies/platform/pentaho-cdf-dd-solution/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>pentaho</groupId>
+    <artifactId>cde-plugin-assemblies-platform</artifactId>
+    <version>8.1.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>pentaho-cdf-dd-samples</artifactId>
+  <version>8.1.0.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <description>Pentaho Community Dashboard Editor plugin samples for the Platform Server</description>
+  <properties>
+    <assembly_appendId>false</assembly_appendId>
+  </properties>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <configuration>
+          <finalName>pentaho-cdf-dd-solution-${project.version}</finalName>
```

Comment:
This as no practical utility, it only changes the filename locally, you can just remove the plugin entry completely. What was the goal of it?

Reply:
The current zip filename is `pentaho-cdf-dd-solution-8.1.0.0-SNAPSHOT.zip` but since the new artifact ID is `pentaho-cdf-dd-samples` the filename will become `pentaho-cdf-dd-samples-8.1.0.0-SNAPSHOT.zip`. I'm not sure if changing the filename will break the script that unzips and loads the samples into the Pentaho Server so I use this plugin to rename it to keep the same filename.